appconfig/appconfig.go

```diff
@@ -35,16 +36,13 @@ func setDefaultParams() {
 	viper.SetDefault("log.show_in_server", false)
 	viper.SetDefault("log.rotation_min", "5")
 	viper.SetDefault("synchronization_service.connection_timeout_seconds", 20)
+	viper.SetDefault("query_log.rotation_min", "5")
```

Comment:
query_log isn't clear enough. It would be better to use: sql_log or sql_debug_log

Reply:
fixed